.github/workflows/downstream.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Downstream Branch Updates
+
+on:
+  schedule:
+    - cron:  '20 7 * * *'
+    # scheduled at 07:00 every Monday and Thursday
+
+  workflow_dispatch:  # click the button on Github repo!
```

Comment:
Is this supposed to be empty?

Reply:
Think so based off all of the examples - https://github.com/aormsby/F-hugo-theme-hello-friend/blob/Working/.github/workflows/wf-fork-sync.yaml